diff --git a/elementor/includes/editor-templates/panel.php b/elementor/includes/editor-templates/panel.php
index 6993e29..0c79fbb 100644
--- a/elementor/includes/editor-templates/panel.php
+++ b/elementor/includes/editor-templates/panel.php
@@ -10,7 +10,8 @@ if ( ! defined( 'ABSPATH' ) ) {
 /**
  * @var Editor $this
  */
-$document = Plugin::$instance->documents->get( $this->get_post_id() );
+$id = \Drupal::routeMatch()->getParameter('node');
+$document = Plugin::$instance->documents->get( $id );
 
 ?>
 <script type="text/template" id="tmpl-elementor-panel">
@@ -27,7 +28,7 @@ $document = Plugin::$instance->documents->get( $this->get_post_id() );
 	<div id="elementor-panel-page-menu-content"></div>
 	<div id="elementor-panel-page-menu-footer">
 		<a href="<?php echo esc_url_elementor_adapter( $document->get_exit_to_dashboard_url() ); ?>" id="elementor-panel-exit-to-dashboard" class="elementor-button elementor-button-default">
-			<i class="fa fa-wordpress"></i>
+			<i class="fa fa-drupal"></i>
 			<?php echo ___elementor_adapter( 'Exit To Dashboard', 'elementor' ); ?>
 		</a>
 	</div>
diff --git a/elementor/includes/utils.php b/elementor/includes/utils.php
index 1d39cd1..4ef60f0 100644
--- a/elementor/includes/utils.php
+++ b/elementor/includes/utils.php
@@ -590,7 +590,7 @@ class Utils {
 	 * @return bool True is templates are supported, False otherwise.
 	 */
 	public static function is_cpt_custom_templates_supported() {
-		require_once ABSPATH . '/wp-admin/includes/theme.php';
+		// require_once ABSPATH . '/wp-admin/includes/theme.php';
 
 		return method_exists( wp_get_theme_elementor_adapter(), 'get_post_templates' );
 	}
